command-line-interface.md

```diff
@@ -0,0 +1,246 @@
+# Operations
+
+The [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
As with the above description, I think it's important to say something like "for the protocols/apis it implements" as a qualification. Then at the title of the cli-api spec say something like the requirements specified in this section/document/specification are for implementations of the cli-api spec.  Without some header to the document anyone reading this document does not know which one is the super the readme or this one.